### PR TITLE
Redact credentials from invoke console output

### PR DESCRIPTION
**Pull request by anshikg**
*Issue #, if available:*

*Description of changes:* When calling cfn invoke locally, the temporary credentials are printed out of the console. This could be a potential security issue. Redacted credentials from the console output. Following is the output when I run cfn invoke on my machine:

```
(env) 3c22fbb25c23:typescript anshikg$ cfn invoke CREATE create.json 
Resource schema is valid.
=== Handler input ===
{
  "requestData": {
    "callerCredentials": "<redacted>",
    "resourceProperties": {
      "Color": "000",
      "Name": "000"
    },
    "previousResourceProperties": {},
    "logicalResourceId": "2b62c178-dd3d-453e-aa67-1f0eb259b1ab"
  },
  "region": "us-east-1",
  "awsAccountId": "594820947549",
  "action": "CREATE",
  "callbackContext": null,
  "bearerToken": "2b62c178-dd3d-453e-aa67-1f0eb259b1ab"
}
=== Handler response ===
{
  "message": "",
  "callbackDelaySeconds": 0,
  "status": "SUCCESS",
  "resourceModel": {
    "Id": "4eb2db78",
    "Name": "000",
    "Color": "000"
  }
}

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by omkhegde — APPROVED** [review]

File: src/rpdk/core/invoke.py [hunk 1]
```diff
@@ -47,7 +47,11 @@ def invoke(args):
     try:
         while _needs_reinvocation(args.max_reinvoke, current_invocation):
             print("=== Handler input ===")
-            print(json.dumps({**payload, "credentials": "<redacted>"}, indent=2))
+
+            payload_copy = payload.copy()
+            payload_copy["requestData"]["callerCredentials"] = "<redacted>"
```
Comment (wbingli) [inline src/rpdk/core/invoke.py:52]: I think we have a  list of loggable properties instead of removing some. Can you use the same approach ?

Comment (anshikg) [reply src/rpdk/core/invoke.py:52]: Could you guys send me a link to the loggable properties.

Comment (wbingli) [reply src/rpdk/core/invoke.py:52]: https://github.com/aws-cloudformation/cloudformation-cli/blob/master/src/rpdk/core/contract/resource_client.py#L588-L600

Comment (anshikg) [reply src/rpdk/core/invoke.py:52]: invoke is calling: https://github.com/aws-cloudformation/cloudformation-cli/blob/master/src/rpdk/core/contract/resource_client.py#L563 which in turn calls the above method. Why do we want to construct the model again?

Comment (wbingli) [reply src/rpdk/core/invoke.py:52]: It's like allow-list vs disallow-list. Allow-list is safer that if any new property introduced , it won't show up in the loggable propeties unless it added.

**Review by wbingli — APPROVED** [review]

**Pull request merged**